.circleci/docker/common/install_openssl.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -ex
+
+OPENSSL=openssl-1.1.1k
+
+wget -q -O "${OPENSSL}.tar.gz" "https://www.openssl.org/source/${OPENSSL}.tar.gz"
```

Comment:
Can you add a comment why we prefer to download those from openssl.org rather than use tags in github.com?

Reply:
I just copied this from the original PR in `pytorch/builder` that @pbelevich had submitted.

Do we have any particular reason @pbelevich?

Reply:
PR for reference, https://github.com/pytorch/builder/pull/708

Reply:
> Can you add a comment why we prefer to download those from openssl.org rather than use tags in github.com?

Because the source code from openssl.org is official with official checksum, and source code from github.com is  just a github's exported source code which is not authorized by anybody